src/init/system_start_service.c

```diff
@@ -32,6 +43,7 @@ int system_start(void) {
 	setup_tty(OPTION_STRING_GET(tty_dev));
 
 	array_foreach(command, script_commands, ARRAY_SIZE(script_commands)) {
+		MD(printf(">%s\n", command));
```

Comment:
May be `log_boot()` can be used here?

Reply:
No, I don't think so. it's a first user process, it should use printf in my mind